clients/spark-extensions-base/src/test/java/org/projectnessie/spark/extensions/AbstractSparkSqlTest.java

```diff
@@ -140,7 +141,7 @@ protected static void assertEquals(
   }
 
   @FormatMethod
-  protected static List<Object[]> sql(String query, Object... args) {
+  protected static List<Object[]> sql(@FormatString String query, Object... args) {
```

Comment:
nit: i think the first string in the method signature is automatically assumed to be the FormatString by FormatMethod, so this could have been redundant

Reply:
yep